CHANGELOG-2.2.md

```diff
@@ -1,3 +1,26 @@
+# Release notes for v2.2.1
+# Changelog since v2.2.0
+
+## Changes by Kind
```

Comment:
We're missing the note about the metrics?

Reply:
The notes is included in the changed section where you can see the csi-lib-utils lib upgrade.
I am not sure if we want to separately add another note here.

Reply:
From the changelog it's not really clear *why* we needed a new release. Is it just the metrics? Or release-tools update?

Reply:
The intention is the metric fix. But it is already bound into the release-tools update.